### PR TITLE
Added security configuration for project, added CustomGlobalException…

### PR DESCRIPTION
**Pull request by vicire**
…Handler

### CODE REVIEW

File: src/main/java/cinema/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                .withUser("bohdan@gmail.com")
+                .password(getEncoder().encode("12345")).roles("ADMIN");
+    }
+
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+                .anyRequest().authenticated()
+                .and()
+                .formLogin()
+                .permitAll()
```
Comment (Max-Pochepets) [inline src/main/java/cinema/config/SecurityConfig.java]:
```suggestion
                .formLogin().permitAll()
```
keep permission on the same level with endpoints

File: src/main/java/cinema/controller/CustomGlobalExceptionHandler.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+public class CustomGlobalExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(
+            MethodArgumentNotValidException ex, HttpHeaders headers,
+            HttpStatus status, WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", new Date());
+        body.put("status", status.value());
+        List<String> errors = ex.getBindingResult()
+                .getFieldErrors()
```
Comment (Max-Pochepets) [inline src/main/java/cinema/controller/CustomGlobalExceptionHandler.java]: Try to test the case when password don't match. This wouldn't work

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]

File: src/main/java/cinema/config/SecurityConfig.java (hunk 1 above)
Comment (vicire) [reply src/main/java/cinema/config/SecurityConfig.java]: Fixed

File: src/main/java/cinema/controller/CustomGlobalExceptionHandler.java (hunk 2 above)
Comment (vicire) [reply src/main/java/cinema/controller/CustomGlobalExceptionHandler.java]: Fixed

**Review by Max-Pochepets — APPROVED** [review]

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**